parts/linux/cloud-init/artifacts/cse_install.sh

```diff
@@ -614,9 +614,11 @@ installKubeletKubectlAndKubeProxy() {
             if [[ ! -z ${BOOTSTRAP_PROFILE_CONTAINER_REGISTRY_SERVER} ]]; then
                 # network isolated cluster
                 echo "Detect Bootstrap profile artifact is Cache, will use oras to pull artifact binary"
-                registry_url="${BOOTSTRAP_PROFILE_CONTAINER_REGISTRY_SERVER}/${K8S_REGISTRY_REPO}/kubernetes-node:v${KUBERNETES_VERSION}-linux-${CPU_ARCH}"
+                KUBE_BINARY_REGISTRY_URL=""
```

Comment:
this line shouldn't be needed

Reply:
to be safe you can use `"${KUBE_BINARY_REGISTRY_URL:-}"` in the call to extractKubeBinaries on line 621

Reply:
fixed